### ISSUE TITLE
Generated docs include super class' methods

### ISSUE THREAD

**Issue by ki4070ma** [timothycrosley/portray#48]
### Problem

Generated docs include super class' methods.

<img width="1148" alt="methods" src="https://user-images.githubusercontent.com/1549855/65815032-cbf45c00-e224-11e9-93e4-c4ca26d97604.png">

### Expected results
Generated docs include methods only for the target class.

https://github.com/ki4070ma/python-client/blob/master/appium/webdriver/extensions/device_time.py
 - device_time
 - get_device_time

### Used portray version
1.3.1

### Generated docs
https://ki4070ma.github.io/python-client/reference/appium/webdriver/extensions/device_time/

**Comment by kootenpv** [comment]
I agree - I would like an option to prevent creating documentation for super level too.

In my case, I inherit from a pandas DataFrame and that leads to way too much documentation.

**Comment by rachel-1** [comment]
Is this a planned feature? It looks like this functionality is now available in the original pdoc repo: https://github.com/mitmproxy/pdoc/issues/15#issuecomment-762951529. In fact, the pdoc repo is now actively maintained and has [an example for running with mkdocs](https://github.com/mitmproxy/pdoc/tree/main/examples/mkdocs)...I like portray for its simplicity, but I'm working with code that subclasses Pytorch's `nn.module` so the inability to hide the huge list of methods from the parent class would be a deal-breaker.
